Dockerfile

```diff
@@ -1,18 +1,27 @@
 FROM python:3.12-slim-bookworm
 
+COPY --from=ghcr.io/astral-sh/uv:0.8.21 /uv /uvx /bin/
+
 WORKDIR /app
 
-RUN apt-get update && apt-get install -y curl
+# Install system dependencies and uv
+RUN apt-get update && apt-get install -y curl && \
+  curl -LsSf https://astral.sh/uv/install.sh | sh && \
```

Comment:
I had to add line 3 to get uv in path and didn't remove this afterwards so it _might_ be safe to remove this curl.